3party/opening_hours/opening_hours_tests/opening_hours_tests.pro

```diff
@@ -0,0 +1,17 @@
+TARGET = opening_hours_tests
+CONFIG += console worn_off
+CONFIG -= app_bundle
+TEMPLATE = app
+
+ROOT_DIR = ../../..
+DEPENDENCIES += opening_hours \
+
+include($$ROOT_DIR/common.pri)
+
+OPENING_HOURS_INCLUDE = $$ROOT_DIR/3party/opening_hours
+INCLUDEPATH += $$OPENING_HOURS_INCLUDE
+
+SOURCES += osm_time_range_tests.cpp
+HEADERS += $$OPENING_HOURS_INCLUDE/osm_time_range.hpp \
```

Comment:
Why do we need to explicitly specify headers from other library? Changes in headers must be handled by DEPENDENCIES, isn't it?